src/frontend/package-lock.json

```diff
@@ -4517,7 +4517,7 @@
     "boolbase": {
       "version": "1.0.0",
       "resolved": "https://registry.npmjs.org/boolbase/-/boolbase-1.0.0.tgz",
-      "integrity": "sha1-aN/1++YMUes3cl6p4+0xDcwed24=",
+      "integrity": "sha512-JZOSA7Mo9sNGB8+UjSgzdLtokWAky1zbztM3WRLCbZ70/3cTANmQmOdR7y2g+J0e2WXywy1yS468tY+IruqEww==",
```

Comment:
Is the `boolbase` integrity hash changing related to eventsource somehow? Did boolbase just happen to change from sha1 to sha512 recently and that got scooped up in this PR, or is something else going on?